sim/dex-species.ts

```diff
@@ -481,7 +481,8 @@ export class DexSpecies {
 					species.tier = species.doublesTier = 'Illegal';
 				}
 			}
-			species.nfe = !!(species.evos.length && this.get(species.evos[0]).gen <= this.dex.gen);
+			const legalEvos = species.evos.filter(mon => !this.get(mon).isNonstandard);
+			species.nfe = !!(legalEvos.length && this.get(legalEvos[0]).gen <= this.dex.gen);
```

Comment:
Would `some` be better than messing around with `filter`, something like this?
```suggestion
			species.nfe = species.evos.some(mon => this.get(mon).gen <= this.dex.gen && !this.get(mon).isNonstandard);
```

Reply:
These effectively do the same thing

Reply:
I wouldn't have suggested code if I thought that it didn't calculate `nfe` correctly... the point was just to have nicer-looking code.